python/kfserving/kfserving/models/knative_dev_pkg_apis_condition.py

```diff
@@ -16,10 +16,10 @@
 
 import six
 
-from kfserving.models.knative_volatile_time import KnativeVolatileTime  # noqa: F401,E501
+from kfserving.models.knative_dev_pkg_apis_volatile_time import KnativeDevPkgApisVolatileTime  # noqa: F401,E501
 
 
-class KnativeCondition(object):
+class KnativeDevPkgApisCondition(object):
```

Comment:
should we still keep the name `KnativeCondition`?

Reply:
Agree with you, updated, thanks!